cypress/integration/Fields/PDQCitations/index.js

```diff
@@ -0,0 +1,57 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
+Given('user is navigating to {string}', (a) => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message like 'list not defined' etc.
+        // and don't want to fail the test so we return false
+        if (err.message.includes(`Cannot read property 'getAttribute' of undefined`)) {
+            return false
+        }
+    });
+    cy.visit(a);
+});
+
+Then('pdq citation section is visible', () => {
+    cy.get('div.pdq-sections > ol').should('be.visible');
+});
+
+And('pdq citation section number 1 header text is {string}', (headerText) => {
+    cy.get('div.pdq-sections > h6').first().should('be.visible').and('have.text', headerText);
```

Comment:
since you are now passing the number for section, you should utilize it in your code - 
1) And('pdq citation section number {int} header text is {string}', (num, headerText) => {
2)  cy.get('div.pdq-sections > h6').eq(num-1).should('be.visible').and('have.text', headerText);

Reply:
Done